content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,391 +1,287 @@
 ---
 title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: ASPNETCORE rolldice
 weight: 10
-cSpell:ignore: KHTML loglevel nameof
 ---
 
-OpenTelemetry for .NET is unique among OpenTelemetry implementations, as it is
-integrated with the .NET `System.Diagnostics` library. At a high level, you can
-think of OpenTelemetry for .NET as a bridge between the telemetry available
-through `System.Diagnostics` and the greater OpenTelemetry ecosystem, such as
-OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
+This page will show you how to get started with OpenTelemetry in .NET.
 
-## ASP.NET Core
+You will learn how you can instrument a simple .NET application automatically,
+in such a way that [traces][], [metrics][] and [logs][] are emitted to the
+console.
 
-The following example demonstrates using Instrumentation Libraries and manual
-instrumentation via an ASP.NET Core app.
+## Prerequisites
 
-First, create your basic ASP.NET Core site:
+Ensure that you have the following installed locally:
 
-```shell
-dotnet new mvc
-```
+- [.NET SDK](https://dotnet.microsoft.com/download/dotnet) 6+
 
-Next, Add the Core OpenTelemetry packages
+## Example Application
 
-```shell
-dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting
-```
+The following example uses a basic
+[Minimal API with ASP.NET Core](https://learn.microsoft.com/aspnet/core/tutorials/min-web-api)
+application. If you are not using ASP.NET Core, that's OK — you can still use
+OpenTelemetry .NET Automatic Instrumentation.
 
-Now let's add the instrumentation packages for ASP.NET Core. This will give us
-some automatic spans for each HTTP request to our app.
+For more elaborate examples, see
+[examples](/docs/instrumentation/net/examples/).
 
-```shell
-dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
-```
+### Dependencies
```

Comment:
what does the heading "Dependencies" mean?

Reply:
Leftover from java structure.
Merged with Create and luaunch an HTTP server 323f3cd22e67bbe176e7fc41242d075236bf37c6